pkg/apis/stream/v1alpha1/stream_defaults.go

```diff
@@ -22,6 +22,6 @@ func (s *Stream) SetDefaults(ctx context.Context) {
 	s.Spec.SetDefaults(ctx)
 }
 
-func (spec *StreamSpec) SetDefaults(ctx context.Context) {
-	// TODO
+func (ss *StreamSpec) SetDefaults(ctx context.Context) {
+	ss.ContentType = "application/octet-stream"
```

Comment:
this will overwrite any existing value

Reply:
yep, just noticed it in practice.
Ambiguous docs FTW :(
```
// Defaultable defines an interface for setting the defaults for the
// uninitialized fields of this instance.
type Defaultable interface {
	SetDefaults(context.Context)
}
```